README.md

```diff
@@ -64,6 +64,20 @@ Flavors can be built in parallel easily with the `build.parallel` target:
 make FLAVORS="<flavor> <other flavor> <...>" build.parallel
 ```
 
+### Building with a specific version of Ceph
+Centos builds can select a specific version of Ceph to install. You just have to pad the required version to the ceph release code name.
```

Comment:
Doc should probably be rephrased to not mention centos specifically.  Maybe something like.
```
Builds can specifiy a specific point release version of Ceph to install by appending the package-manager-appropriate version to the ceph version. Only `-` and `=` are supported as specifiers.

E.g., `luminous-12.2.2` (for yum/zypper) or `luminous=12.2.2` (for apt)

The `CEPH_VERSION` will be interpreted during staging into 2 new environment variables:
- `CEPH_VERSION_NAME` will get the ceph version without point release text
- `CEPH_POINT_RELEASE` will get the point release text including the `-` or `=`
```